<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerDetailsPopup</class>
 <widget class="QWidget" name="CustomerDetailsPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Customer Details</string>
  </property>
  <widget class="QFrame" name="container_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>480</width>
     <height>430</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QFrame#container_frame {
       background-color: white;
       border: 2px solid #3855DB;
       border-radius: 20px;
     }
    </string>
   </property>
   <widget class="QFrame" name="title_bar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
      QFrame#title_bar {
        background-color: #3855DB;
        border-top-left-radius: 18px;
        border-top-right-radius: 18px;
      }
     </string>
    </property>
    <widget class="QLabel" name="label_title">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>440</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white; font-size: 18px; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Customer Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_close">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
       QPushButton { background-color: transparent; color: white; font-size: 20px; border: none; }
       QPushButton:hover { background-color: rgba(255,255,255,0.2); border-radius: 15px; }
      </string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QPushButton" name="print">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
            background-color: #d9d9d9; /* Example background color */
            color: #737373; /* Example text color */
            border-radius: 15px; /* Adjust this value for desired roundness */
}
QPushButton:hover {
    background-color: #9c9b9b; /* Color when the mouse hovers over it */

}
QPushButton:pressed {
    background-color: #FFFFFF; /* Color when the button is clicked down */
	color: #9c9b9b;
	border-style: outset;
	border-width: 3px;
    border-radius: 15px; /* Adjust this value for desired roundness */
    border-color: #9c9b9b;
}</string>
     </property>
     <property name="text">
      <string>Print as PDF</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="content_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>461</width>
      <height>361</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_personal_info">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>150</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Fredoka-SemiBold; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Personal Info</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_email">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>35</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Email:</string>
     </property>
    </widget>
    <widget class="QLabel" name="value_email">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>35</y>
       <width>250</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_phone">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Phone:</string>
     </property>
    </widget>
    <widget class="QLabel" name="value_phone">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>250</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_member_since">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>85</y>
       <width>100</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Member Since:</string>
     </property>
    </widget>
    <widget class="QLabel" name="value_member_since">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>85</y>
       <width>200</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_order_stats">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>150</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Fredoka-SemiBold; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Order Statistics</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_total_orders">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>100</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Total Orders:</string>
     </property>
    </widget>
    <widget class="QLabel" name="value_total_orders">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_total_spent">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>175</y>
       <width>100</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Total Spent:</string>
     </property>
    </widget>
    <widget class="QLabel" name="value_total_spent">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>175</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_last_order">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>100</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Last Order:</string>
     </property>
    </widget>
    <widget class="QLabel" name="value_last_order">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>200</y>
       <width>250</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_recent_order">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>150</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Fredoka-SemiBold; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Recent Order:</string>
     </property>
    </widget>
    <widget class="QLabel" name="value_recent_activity">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>265</y>
       <width>440</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_order_history">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>315</y>
       <width>150</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Fredoka-SemiBold; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Addresses:</string>
     </property>
    </widget>
    <widget class="QLabel" name="value_Addresses">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>441</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../images/Resc_rc.qrc"/>
 </resources>
 <connections/>
</ui>
